src/Orleans.Core/DistributedTracing/EventSourceEvents.cs

```diff
@@ -0,0 +1,71 @@
+using System;
+using System.Collections.Generic;
+using System.Diagnostics.Tracing;
+using System.Text;
+
+namespace Orleans.DistributedTracing.EventSourceEvents
```

Comment:
IMO the namespace should be `Orleans` or `Orleans.Runtime` & the classes should all be marked `internal sealed`